FWCore/PythonUtilities/python/LumiList.py

```diff
@@ -242,7 +246,7 @@ def getLumis(self):
         Return the list of pairs representation
         """
         theList = []
-        runs = self.compactList.keys()
+        runs = list(self.compactList.keys())
         runs = sorted(run, key=int)
```

Comment:
(possibly not for this PR) I wonder if this should be
```python
runs = sorted(runs, key=int)
```
given that there is no local variable `run`.